src/main/java/CommandBye.java

```diff
@@ -0,0 +1,19 @@
+import java.io.IOException;
+
+public class CommandBye extends Command {
+
+    @Override
+    public void run(TaskList tasks, Storage storage, UI ui) {
+        try {
+            storage.storeFile(tasks.getTasks());
```

Comment:
I believe the requirement is to save the changes whenever the task list is changed.